### PR TITLE
CLIC_o3_v11 minor corrections 

### PR DESCRIPTION
**Pull request by andresailer**
BEGINRELEASENOTES
- CLIC_o3_v11: move HCal_cell_size constant to main file to avoid duplicate definition
- CLIC_o3_v11: BeamCal: set correct size of incoming beampipe, only changes of absorber shape, reduce Beampipe wall thickness in beamcal by 0.02 mm to avoid overlaps

ENDRELEASENOTES

### CODE REVIEW

File: CLIC/compact/CLIC_o3_v11/BeamCal_o1_v01_01.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
                         <shape type="Box"  dx="world_side" dy="world_side" dz="world_side"/>
                         <shape type="BooleanShape" operation="Subtraction">
                             <shape type="Tube"  rmin="BeamCal_inner_radius" rmax="BeamCal_outer_radius+env_safety" dz="BeamCal_dz+env_safety"/>
-                            <shape type="Tube"  rmin="0" rmax="3.5*mm" dz="BeamCal_dz+10*env_safety"/>
+                            <shape type="Tube"  rmin="0" rmax="3.7*mm" dz="BeamCal_dz+10*env_safety"/>
```
Comment (petricm) [inline CLIC/compact/CLIC_o3_v11/BeamCal_o1_v01_01.xml]: Wouldn't it be better to define this variable in the global scope with a sensible name and then use in `Beampipe_o1_vo1_02.xml` and here. This is kind of a free floating number, and with time we forget what this is.

File: CLIC/compact/CLIC_o3_v11/HCalBarrel_o1_v01_01.xml [hunk 2]
```diff
@@ -3,7 +3,6 @@
     <define>
         <constant name="HCalBarrel_layers" value="(int) 60"/>
         <constant name="HCalBarrel_layer_thickness" value="2.0*cm + 0.65*cm"/>
-        <constant name="HCal_cell_size" value="3.0*cm"/>
```
Comment (petricm) [inline CLIC/compact/CLIC_o3_v11/HCalBarrel_o1_v01_01.xml]: Please do this also for all other cell sizes.

Comment (andresailer) [inline CLIC/compact/CLIC_o3_v11/HCalBarrel_o1_v01_01.xml]: Other cell sizes are not defined more than once though

File: CLIC/compact/CLIC_o3_v11/BeamCal_o1_v01_01.xml (hunk 1 above)
Comment (andresailer) [inline CLIC/compact/CLIC_o3_v11/BeamCal_o1_v01_01.xml]: Yes, done.

File: CLIC/compact/CLIC_o3_v11/HCalBarrel_o1_v01_01.xml (hunk 2 above)
Comment (andresailer) [inline CLIC/compact/CLIC_o3_v11/HCalBarrel_o1_v01_01.xml]: Done

**Pull request merged**